core-model/src/main/java/com/sequenceiq/cloudbreak/domain/ClusterDefinition.java

```diff
@@ -22,8 +24,9 @@
 import com.sequenceiq.cloudbreak.domain.workspace.WorkspaceAwareResource;
 
 @Entity
+@Where(clause = "archived = false")
```

Comment:
We should watch for archived=false and status=DEFAULT to be returned for some of the repo methods.